carto-package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "postgis",
+    "current_version": {
+        "requires": {
+            "postgres": ">=10.1.1"
```

Comment:
Technically this release (2.4) only requires Postgresql 9.4.

Reply:
I know, you should look into the other issue to have a little bit more context.

The question here is whether our on-premise *dependency system* will support 9.4, not what our fork can actually support.

Reply:
Well, I think that we should stick not to the versions we have but represent the reality of the piece, I'd put >=9.4